web/admin/src/ContentEditor.js

```diff
@@ -28,6 +36,74 @@ export default class ContentEditor extends PureComponent {
     )
   }
 
+  renderTable = () => {
+    var list = this.props.list
+    if (this.state.search){
+      list = this.state.newList
+    }
+    return (
+      <div>
+        <p>Who should receive this content?</p>
+        <SearchBar updateList={this.updateList}/>
+        <span className="leftContainer">
+          <ul className="formBox">{ list.map(c => (
```

Comment:
`map` here is taking the function `c => this.selectAttendee(c)` which is a function with one argument (`c`) which simply calls `this.selectAttendee` with that one argument.  Thus, you can remove the extra function call: `list.map(this.selectAttendee)`